pytest_ibutsu.py

```diff
@@ -141,7 +141,7 @@ class IbutsuArchiver(object):
     frontend = None
 
     def __init__(self, source=None, path=None, extra_data=None):
-        self._results = {}
+        self.results = {}
```

Comment:
Pretty sure I make sure of `_results` in platform-ui - commenting so I don't forget to update it down the line :smile: